services/userlog/pkg/service/templates.go

```diff
@@ -54,6 +54,11 @@ var (
 		Subject: Template("Share expired"),
 		Message: Template("Access to {resource} expired"),
 	}
+
+	PlatformDeprovision = NotificationTemplate{
+		Subject: Template("Instance will be shut down and deprovisioned"),
```

Comment:
Shockmessage, should be more softer IMHO like "Instance will be shutdown in %d days|months"

/cc @mmattel

Reply:
:+1: good point. Will open new PR for it because of the current CI issues